app/src/main/java/com/chess/clock/engine/StageManager.java

```diff
@@ -228,10 +226,11 @@ public boolean isEqual(StageManager sm) {
      * Performs a chess move in the current stage.
      */
     public void addMove() {
+        Stage currentStage = getCurrentStage();
+        if (currentStage == null) return;
```

Comment:
Is it possible for stage to be null during addMove?

Reply:
Right, probably not, will update this 👍